docs/src/main/sphinx/functions/json.rst

```diff
@@ -1450,10 +1450,12 @@ the following requirements are met:
     Cast operations with supported :ref:`character string types
     <string-data-types>` treat the input as a string, not validated as JSON.
     This means that a cast operation with a string-type input of invalid JSON
```

Comment:
i'd remove "of invalid JSON" or add "in particular"  or "for example" to convey this is just example, not a rule

Reply:
I think talking about invalid json just confuses things. There’s nothing special about invalid json, xml, or any random string in this case, so by mentioning it here it creates a relationship that doesn’t exist.

I would just say that it converts a sql string to a json string and leave it at that.